.github/workflows/tests.yml

```diff
@@ -0,0 +1,63 @@
+name: Test package
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.type }}
+  cancel-in-progress: true
+
+on:
+  push:
+
+jobs:
+    packages:
+        name: Test on ${{ matrix.os }} (${{ matrix.python }})
+        runs-on: ${{ matrix.os }}
+        strategy:
+          fail-fast: true
+          matrix:
+            os: [ubuntu-latest, macos-latest, windows-latest]
+            python: ["3.6", "3.7", "3.8", "3.9", "3.x"]
```

Comment:
i think we should add 3.10 and remove 3.6, which is EOL

Reply:
oh, wait, is 3.x == 3.10?

Reply:
3.x is the latest "minor" version of python 3, currently, this is 3.10, but will change to 3.11 as soon as it's released.
This way, if a new version breaks stuff, it is noted right away.

I will remove 3.6, no need to support it with EOL exceeded.